python/mujinplanningclient/realtimeitl3planningclient.py

```diff
@@ -38,9 +65,35 @@ def __init__(self, **kwargs):
             callerid (str, optional): Caller identifier to send to server on every command
             ignoredArgs: Additional keyword args are not used, but allowed for easy initialization from a dictionary
         """
-        super(RealtimeITL3PlanningClient, self).__init__(tasktype='realtimeitlplanning3', **kwargs)
+        
+        super(RealtimeITL3PlanningClient, self).__init__(
+            robotname=robotname,
+            robotspeed=robotspeed,
+            robotaccelmult=robotaccelmult,
+            envclearance=envclearance,
+            robotBridgeConnectionInfo=robotBridgeConnectionInfo,
+            taskzmqport=taskzmqport,
+            taskheartbeatport=taskheartbeatport,
+            taskheartbeattimeout=taskheartbeattimeout,
+            tasktype=tasktype,
```

Comment:
Let's hard-code this in the class as well. The "task" determines the API that the client talks to, so it makes no sense to change it. The user shouldn't have the option or be confused by it.

Reply:
Done